distributed/utils.py

```diff
@@ -29,6 +30,12 @@
 logger = logging.getLogger(__name__)
 
 
+if PY3 and not sys.platform.startswith('win'):
+    mp_context = multiprocessing.get_context('forkserver')
```

Comment:
It might be nice to get this from the config file so that we can set this to fork for faster tests during local development.

```python
from .config import config
context = config.get('multiprocessing_context', 'forkserver')
mp_context = multiprocessing.get_context(context)
```

There is probably a better name for this.